java/ql/lib/semmle/code/java/frameworks/android/Android.qll

```diff
@@ -27,7 +27,12 @@ class AndroidComponent extends Class {
 
   /** The XML element corresponding to this Android component. */
   AndroidComponentXmlElement getAndroidComponentXmlElement() {
-    result.getResolvedComponentName() = this.getQualifiedName()
+    // Find an element with an identifier matching the qualified name of the component.
+    // Aliases have two identifiers (name and target), so check both identifiers (if present).
```

Comment:
Isn't the `target` attribute the one which actually points to the activity being aliased? AFAIK the alias' `name` can be arbitrarily set and doesn't affect its actual target.